distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Result_Set.enso

```diff
@@ -133,7 +130,7 @@ type Builder
 
        Argument:
        - name: The name of the column.
-    make_column : Text -> Java_Exports.Column
+    make_column : Text -> Materialized_Column.Column
```

Comment:
I think this change is inaccurate.

The method in `Java_Exports` creates a Java class `org.enso.table.data.table.Column` - that is a different entity from the Enso `Materialized_Column.Column`.

Reply:
No have changed the `Java_Exports` module to also return an Enso `Column` as could then just use the `Table.new` function.

Reply:
Ah, sorry! I did not notice that.

Shall we adapt `make_table_without_columns` to also return an Enso Table? Otherwise I think it can be confusing if one method from this module returns a raw Java class and other the Enso wrapper and it's impossible to tell from the method name which does which.

Reply:
Done.